README.md

```diff
@@ -419,6 +419,71 @@ def generate_launch_description():
 }
 ```
 
+Here's an example YAML configuration for both composable player and recorder:
+```yaml
+recorder:
+  ros__parameters:
+    use_sim_time: false
+    record:
+      all: true
+      is_discovery_disabled: false
+      rmw_serialization_format: "cdr"
+      topic_polling_interval:
+        sec: 0
+        nsec: 10000000
+      compression_mode: "file"
+      compression_format: "zstd"
+      compression_queue_size: 50
+      compression_threads: 4
+      include_hidden_topics: true
+      include_unpublished_topics: false
+      ignore_leaf_topics: false
+      start_paused: false
```

Comment:
I would reduce the number of parameters in the example YAML file.  
For instance, I would remove parameters related to compression and serialization at least
```suggestion
    record:
      all: true
      is_discovery_disabled: false
      topic_polling_interval:
        sec: 0
        nsec: 10000000
      include_hidden_topics: true
      ignore_leaf_topics: false
      start_paused: false
```